### DISCUSSION TITLE
Welcome to JDA Discussions!

### DISCUSSION THREAD

**Discussion by DV8FromTheWorld** [discord-jda/JDA#1537] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about or get help
    * The [Questions and Help](https://github.com/DV8FromTheWorld/JDA/discussions/categories/questions-and-help) category is meant just for this! Think of it as JDA-specific stackoverflow.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

The intention here is to provide an asynchronous space for our community to communicate with eachother that is outside of Discord. Discord is a fantastic tool for real-time communication and help, but it can struggle to:
 - maintain easily referenceable answers to common questions
 - keep up with when the chat moves quickly and users are busy
 - have async conversations about a specific topic.
   - Often the fast pace chat in Discord can make it difficult to have long-running conversations about a topic, especially if it isn't the main topic in the chat at the time. 

As such, the intention is for Github Discussions to act as a good drop-in to handle these use cases.

**Comment by Andre601** [comment]
Perhaps pinning it would be good, so that it's always findable when there are multiple posts.

**Reply by DV8FromTheWorld** [reply]
Good idea. I'll do that for now.
